src/test/resources/xmlOverrideNamespaces.ds

```diff
@@ -0,0 +1,6 @@
+/** DataSonnet
+version=1.0
+dataformat.application/xml.output.NamespaceDeclarations.c=http://example.com/1
+dataformat.application/xml.output.NamespaceDeclarations.=http://example.com/2
```

Comment:
Hrmm, this syntax is a bit awkward; we should definitely keep it, but it looks like we should thoroughly document the possibilities, plus probably restrict the allowed data types in data format parameters quite a bit